### PR TITLE
Add cbn_cli script and corresponding tests

### PR DESCRIPTION
**Pull request by krishnasantosh**
Command line tool to interact with Chronicle's Config Based Normalizer APIs.

### CODE REVIEW

File: cbn_cli.py
Comment (vkoppaka) [inline cbn_cli.py:44]: Can we remove this?

File: README.md
Comment (vkoppaka) [inline README.md:66]: Can we add a sample command for retrieving errors?

File: cbn_cli.py
Comment (daabr) [reply cbn_cli.py:44]: I agree, and I also think we should change or get rid of get_connecting_url(), although I don't care if it's done in this PR or not. But just to clarify, do you mean you want the user to pass the URL as a command-line arg instead of a string describing the URL (e.g. get_connecting_url() gets "EUROPE" and returns the European URL), or do you have something else in mind?

Comment (krishnasantosh) [reply cbn_cli.py]: I think what venkata whats is to remove the test URL for now.

Regarding remiving get_connecting_url let me remove that in another PR.

File: README.md
Comment (krishnasantosh) [reply README.md:71]: Done

**Comment by krishnasantosh** [comment]
Addressed the comments.

File: cbn_cli.py
Comment (vkoppaka) [reply cbn_cli.py]: What Krishna said (comment just ended up on block of lines as opposed to one). I wanted to remove TEST_URL. I would like to keep get_connecting_url as it serves a purpose.

**Comment by vkoppaka** [comment]
LGTM. @daabr Let us know if you want us to wait for your LGTM as well or if we are OK to merge now.

**Review by daabr — APPROVED** [review]

**Pull request merged**
